form.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <title>Assignment 4</title>
+    <meta charset="utf-8">
+    <script src="http://code.jquery.com/jquery-1.6.4.min.js"></script>
+    <link rel="stylesheet" href="https://code.jquery.com/mobile/1.4.5/jquery.mobile-1.4.5.min.css" />
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <script src="https://code.jquery.com/jquery-1.11.1.min.js"></script>
+    <script src="https://code.jquery.com/mobile/1.4.5/jquery.mobile-1.4.5.min.js"></script>
+    <link rel="stylesheet" href="jquery-mobile-theme-191316-0/themes/ctheme1">
+</head>
+
+<body>
+    <!--    
+        Collin Keating
+        assignment4.html
+        INFO2439.PLA
+        Thoendel
+        9/6/19
+    -->
+	<div data-role="page" id="homePage" data-title="Keating | INFO 2439 | Assignment #2" data-add-back-btn="true" data-back-btn-text="Back">
+<div data-role="header" data-position="fixed"><a data-transition="slide" href="#" data-direction="reverse" class="ui-btn-left ui-btn ui-btn-inline ui-mini ui-corner-all ui-btn-icon-right ui-icon-back">Back</a></div>
+    <div data-role="homePage" data-title="Keating | INFO 2439 | Assignment 4">
+        <header data-role="header" data-position="fixed">
+            <a href="../index.html" data-ajax="false" data-rel="external" data-icon="home">Home</a>
+            <h1>Assignment 4</h1>
+        </header>
+        <div data-role="content">
+            <form action="http://lolberding.portfolios.mccinfo.net/info_survey/bistrosurvey_insert.php" method="post">
```

Comment:
In all the test files - you have my script as the form action - but this one where it counts is pointing elsewhere. Bummer!